cypress/e2e/marc/marc-authority/verify-authority-source-facet-option.cy.js

```diff
@@ -0,0 +1,87 @@
+import Permissions from '../../../support/dictionary/permissions';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import MarcAuthorities from '../../../support/fragments/marcAuthority/marcAuthorities';
+import MarcAuthority from '../../../support/fragments/marcAuthority/marcAuthority';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import { JOB_STATUS_NAMES } from '../../../support/constants';
+
+describe('MARC -> MARC Authority', () => {
+  const testData = {
+    searchOption: 'Personal name',
+    searchValue: 'France',
```

Comment:
Could you please use a value which would never return any results? Like random string with letters and digits. "France" is just an example, and during the test run there can actually be records with "France" in the title in the system